docs/administrator.md

```diff
@@ -154,6 +154,20 @@ data:
   pod_antiaffinity_topology_key: "failure-domain.beta.kubernetes.io/zone"
 ```
 
+### Pod Disruption Budget
+
+By default the operator uses a PodDisruptionBudget (PDB) to protect the cluster
+from voluntarily disruptions and hence unwanted DB downtime. The `MinAvailable`
+parameter of the PDB is set to `1` which makes it work for single-node clusters
+or in situations when there is no master (e.g. standby clusters).
```

Comment:
> which makes it work for single-node clusters

it prevents killing masters in single pods clusters.

> situations when there is no master (e.g. standby clusters).

not really necessary here,  I meant with my previous review is that the comment in the code was unnecessarily generic.